EleksTubeHAX/EleksTubeHAX.ino

```diff
@@ -332,14 +332,21 @@ void setupMenu() {
   tfts.setCursor(0, 124, 4);
 }
 
+bool isDimmingTime(uint8_t current_hour) {
+    if (DAY_TIME < NIGHT_TIME)
+      return (current_hour >= NIGHT_TIME) || (current_hour < DAY_TIME);
+    else
+      return (NIGHT_TIME <= current_hour) && (current_hour < DAY_TIME);  
+}
+
```

Comment:
This logic is hard to follow. I'm having a hard time even figuring out whether it's correct.

I suggest a more straight-forward logic:

```suggestion
bool isNightTime(uint8_t current_hour) {
    if (DAY_TIME < NIGHT_TIME) {
      // "Night" spans across midnight
      return (current_hour < DAY_TIME) || (current_hour >= NIGHT_TIME);
    }
    else {
      // "Night" starts after midnight, entirely contained within the day
      return (current_hour >= NIGHT_TIME) && (current_hour < DAY_TIME);  
    }
}

```

I also changed the name of the function to be consistent.